cmd/bee/cmd/init.go

```diff
@@ -23,25 +22,13 @@ func (c *command) initInitCmd() (err error) {
 				return cmd.Help()
 			}
 
-			var logger logging.Logger
-			switch v := strings.ToLower(c.config.GetString(optionNameVerbosity)); v {
-			case "0", "silent":
-				logger = logging.New(ioutil.Discard, 0)
-			case "1", "error":
-				logger = logging.New(cmd.OutOrStdout(), logrus.ErrorLevel)
-			case "2", "warn":
-				logger = logging.New(cmd.OutOrStdout(), logrus.WarnLevel)
-			case "3", "info":
-				logger = logging.New(cmd.OutOrStdout(), logrus.InfoLevel)
-			case "4", "debug":
-				logger = logging.New(cmd.OutOrStdout(), logrus.DebugLevel)
-			case "5", "trace":
-				logger = logging.New(cmd.OutOrStdout(), logrus.TraceLevel)
-			default:
-				return fmt.Errorf("unknown verbosity level %q", v)
-			}
+			res, err := c.configureSigner(cmd, logging.New(ioutil.Discard, 0))
```

Comment:
@paxthemax `configureSigner` uses the passed logger to print information about the process. 

Using `ioutil.Discard` here essentially means we're silencing all possible warnings and other information that is printed there. I think that some of the information is also good for debugging purposes so i wouldn't be in a rush to remove it.

Why not remove the log lines that print the key info in `configureSigner` and just print the keys nicely here once the configuration part is over. Also, you mentioned different types of printouts and I think that for the terminal output we should print the keys with the descriptions of them (like in the first example in the PR description), since this one only prints the raw keys.